lib/topics/state_and_behaviour.rb

```diff
@@ -4,23 +4,23 @@
 module StateAndBehaviour
   # defines class Car
   class Car
-    attr_reader :color, :model, :year, :current_speed
-    # initialize for Car object
-    def initialize(car_praremeters)
-      @year = car_praremeters[:year] || 2015
-      @color = car_praremeters[:color] || 'purple'
-      @model = car_praremeters[:model] || 'skyline'
-      @current_speed = 0
-    end
-
+    attr_accessor :color, :model, :year, :current_speed
     # create new Car object with default parameters
     def self.default_car
       new
     end
 
+    # initialize for Car object
+    def initialize(car_parameters)
```

Comment:
1) Failure:
Car::#initialize#test_0001_expects hash of attributes [/var/www/pivorak/homework2/test/topics/test_state_and_behaviour.rb:26]:
ArgumentError expected but nothing was raised.

15 runs, 18 assertions, 1 failures, 0 errors, 0 skips